pages/success.vue

```diff
@@ -44,7 +44,7 @@ export default {
     ...mapGetters({
       speaker: 'items/speaker',
       recipientEmail: 'items/recipientEmail',
-      days: 'items/daysToDeadline'
+      timerange: 'items/timerange'
```

Comment:
~@tillprochaska @drjakob no, I woul not do that! `timerange` is always `5` whereas `daysToDeadline` is the number of days until the deadline. The deadline is itself based on `timerange`.~

~The consequence of this change would be that you ordered a text-to-audio production yesterday and the deadline is still 5 days from now, just like yesterday.~